mitosheet/src/mito/utils/keyboardShortcuts.tsx

```diff
@@ -126,6 +126,97 @@ export const keyboardShortcuts: KeyboardShorcut[] = [
             keys: ['h']
         },
         action: ActionEnum.OpenFindAndReplace
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['`']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['`']
+        },
+        action: ActionEnum.Set_Format_Default
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['1']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['1']
+        },
+        action: ActionEnum.Set_Format_Number
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['@']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['@']
+        },
+        action: ActionEnum.Set_DateTime_Dtype
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['3']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['3']
+        },
+        action: ActionEnum.Set_DateTime_Dtype
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['4']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['4']
+        },
+        action: ActionEnum.Set_Format_Currency
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['5']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['5']
+        },
+        action: ActionEnum.Set_Format_Percent
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['^']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['^']
+        },
+        action: ActionEnum.Set_Format_Scientific
```

Comment:
I'm a little confused. We don't have `%` registered, and we don't have `6` registered. Why do we sometimes do both, but not always?

Reply:
I'm not sure why the browser does this but I tested locally to find these.